source/python/idetect/interpreter.py

```diff
@@ -0,0 +1,894 @@
+from textacy.spacy_utils import get_main_verbs_of_sent, get_objects_of_verb, get_subjects_of_verb
+from textacy.extract import pos_regex_matches
+import re
+import parsedatetime
+import string
+from spacy.tokens import Token, Span
+from datetime import datetime, timedelta
+from functools import reduce
+
+
+person_reporting_terms = [
```

Comment:
Hmm..good point....I think we should maybe look to move into the database...it may be an area where analysts / someone will want to be able to add / remove / edit.